Runtime/Plugins/SolanaWalletAdapterWebGL/SolanaWalletAdapterWebGL.jslib

```diff
@@ -1,130 +1,163 @@
 mergeInto(LibraryManager.library, {
-    InitWalletAdapter: async function (callback) {
-        const isXnft = Boolean("xnft" in window && window.xnft != undefined && window.xnft.solana != undefined && window.xnft.solana.publicKey != undefined);
-        // Add UnityWalletAdapter from CDN
-        if(window.walletAdapterLib == undefined){
-            console.log("Adding WalletAdapterLib")
-            var script = document.createElement("script");
-            script.src = "https://cdn.jsdelivr.net/gh/magicblock-labs/unity-js-wallet-adapter@main/dist/wallet-adapter-lib.js";
-            document.head.appendChild(script);
-            script.onload = function() {
-                console.log("WalletAdapterLib loaded");
-                Module.dynCall_vi(callback, isXnft);
-            };
-        }else{
-            Module.dynCall_vi(callback, isXnft);
+  InitWalletAdapter: async function (callback, rpcClusterPtr) {
+    const isXnft = Boolean(
+      "xnft" in window &&
+        window.xnft != undefined &&
+        window.xnft.solana != undefined &&
+        window.xnft.solana.publicKey != undefined
+    );
+    window.rpcCluster = UTF8ToString(rpcClusterPtr);
+    // Add UnityWalletAdapter from CDN
+    if (window.walletAdapterLib == undefined) {
+      var script = document.createElement("script");
+      script.src =
+        "https://cdn.jsdelivr.net/gh/magicblock-labs/unity-js-wallet-adapter@main/dist/wallet-adapter-lib.js";
```

Comment:
let's pin the version to 1.2.0 of the js wallet adapter: https://cdn.jsdelivr.net/gh/magicblock-labs/unity-js-wallet-adapter@v1.2.0/dist/wallet-adapter-lib.js